pages/index.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="container">
+    <h1 class="title">Pokemon Familiy</h1>
+    <router-link to="/pokemons/favourites" class="favourites-link"
+      >Go to Favourites</router-link
+    >
```

Comment:
I know eslint format this like this. But, help eslint to format properly. this is hard to read